ui-components/src/headers/StepHeader.scss

```diff
@@ -0,0 +1,23 @@
+.step-header {
```

Comment:
This is close but not quite the right format for styling v2. A good example is `StandardCard.scss`. We should have no nested classes, one root class with the name of the component, the following classes start with that root, and we have a number of css variables underneath that root specific to this component that we anticipate a consumer might want to override (like here maybe background color, color, min width).